src/IHaskell/Convert/LhsToIpynb.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE CPP #-}
```

Comment:
Ah, got it! I scanned through the source for `#endif` or other directives but saw nothing; didn't think of the cabal macros.